### PR TITLE
chore: update ReSpec syntax and error fixes

### PR DESCRIPTION
**Pull request by sidvishnoi**
I've made small focused commits, so they're easier to review and understand. Please squash and merge in the end.
Non-substantial for IPR.

***
<a href="https://pr-preview.s3.amazonaws.com/sidvishnoi/server-timing/pull/70.html" title="Last updated on Apr 30, 2020, 7:49 AM UTC (5497cb2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/server-timing/70/c660d6a...sidvishnoi:5497cb2.html" title="Last updated on Apr 30, 2020, 7:49 AM UTC (5497cb2)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,7 +94,7 @@
 User agents MUST ignore extraneous characters found after a `metric-name` but before the first `server-timing-param` and before the next `server-timing-metric`.
 
 <p data-link-for="PerformanceServerTiming">
-  This specification establishes the server-timing-params for server-timing-param-names <a data-lt="PerformanceServerTiming.duration">dur</a> and <a data-lt="PerformanceServerTiming.description">desc</a>, both optional.
+  This specification establishes the server-timing-params for server-timing-param-names {{duration}} and {{description}}, both optional.
```
Comment (sidvishnoi) [inline index.html]: This should probably be:
```suggestion
  This specification establishes the server-timing-params for server-timing-param-names "dur" for {{duration}} and "desc" for {{description}}, both optional.
```

**Comment by plehegar** [comment]
Marked as non substantive for IPR from ash-nazg.

File: index.html (hunk 1 above)
Comment (plehegar) [reply index.html]: agreed

**Comment by plehegar** [comment]
This is adding "[resource-timing]" in the set of normative references (among other additions). Not quite sure why since we already have resource-timing-2. I guess it doesn't harm anything...

**Comment by sidvishnoi** [comment]
> This is adding "[resource-timing]" in the set of normative references

Updated in https://github.com/w3c/server-timing/pull/70/commits/3c03b8ce8c5b8c631c6d4d292d55d3f5d48c7f9c

> (among other additions)

Earlier, some were linked using `href` (instead of `data-cite`) - so those references were missing.

Question: `current document` is linked to both `navigation-timing-2` as well as `resource-timing-2`. Only one those specs (or some other spec) should be defining it (and we should be linking to that only) - so I think that's a bug.

File: index.html [hunk 2]
```diff
@@ -107,86 +108,86 @@
...
-  The <a>serverTiming</a> attribute returns a sequence of <a>PerformanceServerTiming</a> entries.
+  The {{serverTiming}} attribute returns a sequence of {{PerformanceServerTiming}} entries.
 </section>
 
 <section>
 ## Process
 
 ### Processing Model
-<p>When processing the response of the <a data-cite="NAVIGATION-TIMING-2#current-document">current document</a>, set the <a data-lt="PerformanceResourceTiming.serverTiming">serverTiming</a> attribute on the <a data-cite="NAVIGATION-TIMING-2#step-create-object">newly created</a> `PerformanceNavigationTiming` object to the return value of the <a>server-timing header parsing algorithm</a>
+<p>When processing the response of the <a data-cite="NAVIGATION-TIMING-2#current-document">current document</a>, set the {{PerformanceResourceTiming/serverTiming}} attribute on the <a data-cite="NAVIGATION-TIMING-2#step-create-object">newly created</a> `PerformanceNavigationTiming` object to the return value of the [=server-timing header parsing algorithm=]
```
Comment (sidvishnoi) [inline index.html]:
```suggestion
<p>When processing the response of the <a data-cite="NAVIGATION-TIMING-2#dfn-current-document">current document</a>, set the {{PerformanceResourceTiming/serverTiming}} attribute on the <a data-cite="NAVIGATION-TIMING-2#step-create-object">newly created</a> `PerformanceNavigationTiming` object to the return value of the [=server-timing header parsing algorithm=]
```

**Comment by plehegar** [comment]
We're using "current document" as a short end  for "Window object's newest Document object". I agree that we should only have it once, probably keeping the definition in Resource Timing. Or better, we should stop exporting those definitions since they shouldn't really be used by other specs...

**Comment by sidvishnoi** [comment]
I could not find any direct definition to the term for "Window object's newest Document object". The closest I reached was this: https://www.w3.org/TR/2015/WD-html51-20150831/browsers.html#concept-document-window, which maps to "[associated document](https://html.spec.whatwg.org/multipage/window-object.html#concept-document-window)". Why is the former being used?

**Comment by plehegar** [comment]
I think it's historical at this point. We may be able to drop "current document" entirely and use "associated document" instead. If we're willing to make that change in server-timing, we'll need to file issues against nav- and res- timings.

**Comment by marcoscaceres** [comment]
@plehegar, if you are back, friendly ping 🙏

**Comment by plehegar** [comment]
@marcoscaceres , I'd like a +1 from one of the main editors before making pull requests. Hopefully, this will get noticed next week.

**Comment by sidvishnoi** [comment]
@plehegar Do you want me to open issues in nav- and res- timing specs for use of `"associated document"`? Or this can be merged before we do that?

**Comment by marcoscaceres** [comment]
@yoavweiss, can this be merged? These are all just small syntax changes.

**Comment by yoavweiss** [comment]
Apologies for dropping the ball on reviewing this!

**Pull request merged**